src/plugins/Dashboard/FileItem.js

```diff
@@ -52,16 +54,23 @@ module.exports = function fileItem (props) {
                         }"
                   onclick=${(ev) => {
                     if (isUploaded) return
+                    if (error) {
+                      props.retryUpload(file.id)
+                      return
+                    }
                     if (props.resumableUploads) {
                       props.pauseUpload(file.id)
                     } else {
                       props.cancelUpload(file.id)
                     }
                   }}>
-            ${FileItemProgress({
-              progress: file.progress.percentage,
-              fileID: file.id
-            })}
+            ${props.file.error
```

Comment:
could this also use the `error` variable declared above?

Reply:
Done.